plugins/modules/ipasudocmdgroup.py

```diff
@@ -100,7 +100,8 @@
 """
 
 from ansible.module_utils.ansible_freeipa_module import \
-    IPAAnsibleModule, compare_args_ipa, gen_add_del_lists, ipalib_errors
+    IPAAnsibleModule, compare_args_ipa, gen_add_del_lists, gen_add_list, \
+    gen_intersection_list, ipalib_errors
```

Comment:
The use of gen_add_list and gen_intersection_list to reduce to the needed changes only should also be mentioned. Maybe this could be an extra commit.

Reply:
Without gen_add_list and gen_intersection_list, the patch would be bigger, as a custom `result_handler` would be needed because of the `already a member/not a member` issue.

Do you think it is better to have a commit with the custom handler and another committ/PR removing it?

Reply:
You could add the change with `gen_add_list` and `gen_intersection_list` prior to the other one in the PR for example.

Reply:
Ok. Will do.